lib/common.dart

```diff
@@ -1,6 +1,22 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:jiffy/jiffy.dart';
 
-const DATE_FORMAT = 'yyyy-MM-dd';
+// The date at time formats to use for Jiffy DateTime formatting
+const DISPLAY_DATE_FORMAT = 'yyyy-MM-dd';
+const DISPLAY_TIME_FORMAT = 'HH:mm:ss'; // 24-hour time format
+
+String displayNeatTimestamp(DateTime? dateTime) {
+  // If DateTime is null, set it to a default of 0001-01-01 at 00:00:00
+  if (dateTime == null) {
+    return Jiffy([0001, 01, 01]).format(DISPLAY_DATE_FORMAT) +
+        " at " +
+        Jiffy([0]).format(DISPLAY_TIME_FORMAT);
+  } else {
```

Comment:
There is no value in this code other than to show an error. And we should not build any logic or write any code to account for some hypothetical app errors in the future.